chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.ts

```diff
@@ -212,6 +218,14 @@ export class AutoTabGroupsPageElement extends CrLitElement {
     return loadTimeData.getString('tabOrganizationDismissButtonLabel')
   }
 
+  protected getPrivacyDisclaimerMessage_(): string {
+    return loadTimeData.getString('tabOrganizationPrivacyDisclaimer')
+  }
+
+  protected getEnableButtonLabel_(): string {
+    return loadTimeData.getString('tabOrganizationEnableButtonLabel')
+  }
```

Comment:
out of interest, why do some properties get initialized construct time (i.e. `showFRE`) while others are function results? 

Is it for testing or something?